_posts/en/newsletters/2021-01-27-newsletter.md

```diff
@@ -0,0 +1,129 @@
+  In a follow up to the original 2015 question, Steven Roose provides a one line
+  `bitcoin-cli` command using `getrawtransaction` against a full node to
+  generate a PDF of the Bitcoin whitepaper. [jb55 provides][bitcoin whitepaper gettxout]
+  a similar command using `gettxout` which works against a pruned node.
+
+- [Full list of "special cases" during Bitcoin Script execution?]({{bse}}101142)
+  Pieter Wuille provides an overview of Bitcoin script evaluation in pseudocode
+  including the conditions for additional rules for both [BIP16][] P2SH and
+  [BIP141][] segwit.
+
+- [Would first-seen prevent a double spend attack?]({{bse}}101827)
+  David Lynch asks if moving away from [Replace-By-Fee (RBF)][topic rbf] would
+  prevent double spend attacks. Pieter Wuille describes a variety of nuanced
+  considerations and incentives surrounding transaction propagation on the
+  network concluding that no type of unconfirmed transaction can be trusted and
+  users should wait for confirmations.
+
+- [How do light clients using compact block filters get relevant unconfirmed transactions?]({{bse}}101512)
+  User Pseudonymous explains that while [BIP37][topic transaction bloom
+  filtering] bloom transaction filtering supports unconfirmed transactions,
+  [compact block filters][topic compact block filters] have no such
+  consideration and only service light clients with condensed block data since
+  light clients have no way of verifying that unconfirmed transactions are valid.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.9.3][c-lightning 0.9.3] is the project's newest minor release.
+  It includes several improvements to the user interface and plugin
+  capabilities, plus experimental support for the proposed onion
+  messages protocol (see [Newsletter #92][news92 cl3600]) and the offers protocol (see
+  [Newsletter #128][news128 cl4255]).  See the [release
+  notes][c-lightning 0.9.3] and [changelog][cl cl] for details.
+
+- [LND 0.12.0-beta][] is the latest release for the next major version
+  of this LN software.  It includes support for using watchtowers with
+  [anchor outputs][topic anchor outputs] and adds a new `psbt` wallet
+  subcommand for working with [PSBTs][topic psbt], among other
+  improvements and bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #19866][] adds User Statically Defined Trace probes (USDT) to
```

Comment:
The PR doesn't actually add any USDT probes, just the framework for adding tracepoints. I'll push a commit with minor edits to clarify that.

Reply:
Thanks @jnewbery!

Reply:
Thanks @jnewbery!